index.d.ts

```diff
@@ -142,6 +142,18 @@ declare namespace browserslist {
    */
   function coverage(browsers: readonly string[], stats?: StatsOptions): number
 
+  /**
+   * Lint browserslist config.
+   *
+   * @param [queries=browserslist.defaults] Browser queries.
+   * @param [opts] Browserslist options.
+   * @returns Lint report.
+   */
+  function lint(
+    queries?: string | readonly string[] | null,
+    opts?: browserslist.Options
+  ): string
```

Comment:
I think we need to return `string[]`. We may have multiple errors. We need a simple way to check that there is no error.

Reply:
@ai done